docs/architecture.rst

```diff
@@ -0,0 +1,64 @@
+Architecture
+============
+Here there is info about main components in Superdesk and how these interact.
+To run superdesk we use `honcho <https://github.com/nickstenning/honcho>`_ to define
+processes for each of components:
```

Comment:
You could use [include directive](http://docutils.sourceforge.net/docs/ref/rst/directives.html#including-an-external-document-fragment) here, if not need wrap next strings as code.